Additional kubernetes GPU docs (#2078)
bottlerocket-os/bottlerocket: README.md

* The system kernel will reserve 256MB for the crash kernel, only when the host has at least 2GB of memory; the reserved space won't be available for processes running in the host
* The crash kernel will only be loaded when the `crashkernel` parameter is present in the kernel's cmdline and if there is memory reserved for it

### NVIDIA GPUs Support

> Bottlerocket's `nvidia` kubernetes variants include the required packages and configurations to leverage NVIDIA GPUs.

Reviewer: nit: maybe omit `Kubernetes` since we'll have ECS soon:
```suggestion
Bottlerocket's `nvidia` variants include the required packages and configurations to leverage NVIDIA GPUs.
```

> The official AMIs for these variants can be used with the following EC2 instance types: `p2`, `p3`, `p4`, `g4dn`, `g5` and `g5g`.

Reviewer 2: Suggestion: Guard against having an incorrect list in the future.

```suggestion
The official AMIs for these variants can be used with EC2 GPU-equipped instance types such as: `p2`, `p3`, `p4`, `g4dn`, `g5` and `g5g`.
```

> Please see [QUICKSTART-EKS](QUICKSTART-EKS.md#aws-k8s--nvidia-variants) for further details about kubernetes variants.

Reviewer: ```suggestion
Please see [QUICKSTART-EKS](QUICKSTART-EKS.md#aws-k8s--nvidia-variants) for further details about Kubernetes variants.
```

> Please refer to the [Amazon EKS quickstart](QUICKSTART-EKS.md#aws-k8s--nvidia-variants) for further details about these variants.

Reviewer: Elsewhere in this doc, we refer to this doc as `QUICKSTART-EKS` so I'd like to continue using that name for consistency.
